float.js

```diff
@@ -35,10 +35,13 @@ const getFloatData = function(listingId, inspectLink) {
         return Promise.resolve({ iteminfo: floatData[listingId] });
     }
 
-    return fetch(`https://api.csgofloat.com:1738/?url=${inspectLink}`)
-    .then((response) => {
-        if (response.ok) return response.json();
-        return response.json().then((err) => { throw err; });
+    return new Promise((resolve, reject) => {
+        let thisBrowser = (window.browser) ? browser: chrome;
+
+        thisBrowser.runtime.sendMessage({'inspectLink': inspectLink}, (data) => {
```

Comment:
Sould use `chrome.runtime.sendMessage` here in order to use the callback-style API.
Reference:
> "As a porting aid, the Firefox implementation of WebExtensions supports chrome and callbacks as well as browser and promises. This means that many Chrome extensions will just work in Firefox without any changes. However, this is not part of the WebExtensions standard, and might not be supported by all compliant browsers."
> https://developer.mozilla.org/en-US/Add-ons/WebExtensions/Chrome_incompatibilities#Callbacks_and_the_chrome.*_namespace